### PR TITLE
WPF 890011 - Updated API links for Scheduler tooltip content

### PR DESCRIPTION
**Pull request by TamilarasanGunasekaran**
I have updated the API link for the tooltip and changed the code headings.

### CODE REVIEW

File: wpf/Scheduler/Appointments.md [hunk 1]
```diff
@@ -810,7 +810,7 @@ The following code example shows the usage of DataTemplate.
 </syncfusion:SfScheduler>
 
 N>
-* This property will only be applicable when `EnableTooltip` is set to true.
+* This property will only be applicable when [EnableToolTip](https://help.syncfusion.com/cr/wpf/Syncfusion.UI.Xaml.Scheduler.SfScheduler.html#Syncfusion_UI_Xaml_Scheduler_SfScheduler_EnableToolTip) is set to true.
 
 {% endhighlight %}
```
Comment (Karthickmani97) [inline wpf/Scheduler/Appointments.md:815]: Add API link for `ToolTipTemplate` property

File: wpf/Scheduler/DaysView.md [hunk 2]
```diff
@@ -442,15 +442,15 @@ Customize the default appearance of view header by setting `ViewHeaderTemplate`
 You can customize the spacing between an appointment and the right border of its cell to enhance interaction with a scheduler that contains appointments.
```
Comment (Karthickmani97) [inline wpf/Scheduler/DaysView.md:442]: To enhance interaction with a scheduler's appointments, you can customize the spacing between appointments and the right border of their cells using the `CellRightPadding` property of `DayViewSettings` in the scheduler.

Use above sentence and add API link. Follow same in all places.

Comment (TamilarasanGunasekaran) [reply wpf/Scheduler/DaysView.md]: Updated.

File: wpf/Scheduler/Appointments.md (hunk 1 above)
Comment (TamilarasanGunasekaran) [reply wpf/Scheduler/Appointments.md:815]: Added.

File: wpf/Scheduler/Appointments.md [hunk 3]
```diff
@@ -761,7 +761,7 @@ An interactive tooltip provides additional details about the appointments on hov
 
 ### Enable tooltip for appointments
 
-To enable tooltip for the scheduler appointments, use the `EnableTooltip` property of `SfScheduler.` By default, `EnableTooltip` is set to `false.` To provide users with additional information or context about appointments, simply set this property to true.
+To enable tooltip for the scheduler appointments, use the [EnableToolTip](https://help.syncfusion.com/cr/wpf/Syncfusion.UI.Xaml.Scheduler.SfScheduler.html#Syncfusion_UI_Xaml_Scheduler_SfScheduler_EnableToolTip) property of `SfScheduler.` By default, [EnableToolTip](https://help.syncfusion.com/cr/wpf/Syncfusion.UI.Xaml.Scheduler.SfScheduler.html#Syncfusion_UI_Xaml_Scheduler_SfScheduler_EnableToolTip) is set to `false.` To provide users with additional information or context about appointments, simply set this property to true.
```
Comment (Karthickmani97) [inline wpf/Scheduler/Appointments.md:764]: Add API links for SfScheduler - https://help.syncfusion.com/cr/wpf/Syncfusion.UI.Xaml.Scheduler.SfScheduler.html

File: wpf/Scheduler/DaysView.md [hunk 4]
```diff
@@ -439,18 +439,18 @@ Customize the default appearance of view header by setting `ViewHeaderTemplate`
 
 ## Cell right padding
 
-You can customize the spacing between an appointment and the right border of its cell to enhance interaction with a scheduler that contains appointments.
+To enhance interaction with a scheduler's appointments, you can customize the spacing between appointments and the right border of their cells using the [CellRightPadding](https://help.syncfusion.com/cr/wpf/Syncfusion.UI.Xaml.Scheduler.DaysViewSettings.html#Syncfusion_UI_Xaml_Scheduler_DaysViewSettings_CellRightPadding) property of [DayViewSettings](https://help.syncfusion.com/cr/wpf/Syncfusion.UI.Xaml.Scheduler.DaysViewSettings.html) in the scheduler.
```
Comment (Karthickmani97) [inline wpf/Scheduler/DaysView.md:442]: Replace scheduler to - [SfScheduler](https://help.syncfusion.com/cr/wpf/Syncfusion.UI.Xaml.Scheduler.SfScheduler.html)

File: wpf/Scheduler/Month-View.md [hunk 5]
```diff
@@ -545,18 +545,18 @@ You can customize the default appearance of more appointments indicator in a mon
 
 ## Cell right padding
 
-You can customize the spacing between an appointment and the right border of its cell to enhance interaction with a scheduler that contains appointments.
+To enhance interaction with a scheduler's appointments, you can customize the spacing between appointments and the right border of their cells using the [CellRightPadding](https://help.syncfusion.com/cr/wpf/Syncfusion.UI.Xaml.Scheduler.MonthViewSettings.html#Syncfusion_UI_Xaml_Scheduler_MonthViewSettings_CellRightPadding) property of [MonthViewSettings](https://help.syncfusion.com/cr/wpf/Syncfusion.UI.Xaml.Scheduler.MonthViewSettings.html) in the scheduler.
```
Comment (Karthickmani97) [inline wpf/Scheduler/Month-View.md:548]: Replace scheduler to - [SfScheduler](https://help.syncfusion.com/cr/wpf/Syncfusion.UI.Xaml.Scheduler.SfScheduler.html)

File: wpf/Scheduler/TimelineView.md [hunk 6]
```diff
@@ -450,18 +450,18 @@ Schedule.TimelineViewSettings.TimelineAppointmentHeight = 100;
 
 ## Cell bottom padding
 
-You can customize the spacing between an appointment and the bottom border of its cell to enhance interaction with a scheduler that contains appointments.
+To enhance interaction with a scheduler's appointments, you can customize the spacing between appointments and the bottom border of their cells using the [CellBottomPadding](https://help.syncfusion.com/cr/wpf/Syncfusion.UI.Xaml.Scheduler.TimelineViewSettings.html#Syncfusion_UI_Xaml_Scheduler_TimelineViewSettings_CellBottomPadding) property of [TimelineViewSettings](https://help.syncfusion.com/cr/wpf/Syncfusion.UI.Xaml.Scheduler.TimelineViewSettings.html) in the scheduler.
```
Comment (Karthickmani97) [inline wpf/Scheduler/TimelineView.md:453]: Replace scheduler to - [SfScheduler](https://help.syncfusion.com/cr/wpf/Syncfusion.UI.Xaml.Scheduler.SfScheduler.html)

**Review by Karthickmani97 — COMMENTED** [review]
@TamilarasanGunasekaran , Address my review concern.

File: wpf/Scheduler/Appointments.md (hunk 3 above)
Comment (TamilarasanGunasekaran) [reply wpf/Scheduler/Appointments.md:764]: Added.

File: wpf/Scheduler/DaysView.md (hunk 4 above)
Comment (TamilarasanGunasekaran) [reply wpf/Scheduler/DaysView.md]: Changed

File: wpf/Scheduler/Month-View.md (hunk 5 above)
Comment (TamilarasanGunasekaran) [reply wpf/Scheduler/Month-View.md]: Changed.

File: wpf/Scheduler/TimelineView.md (hunk 6 above)
Comment (TamilarasanGunasekaran) [reply wpf/Scheduler/TimelineView.md]: Changed.

**Review by Karthickmani97 — APPROVED** [review]
This is fine.

**Review by Jeyasri-Murugan — APPROVED** [review]

**Pull request merged**